apps/web/components/apps/OmniInstallAppButton.tsx

```diff
@@ -18,11 +18,13 @@ export default function OmniInstallAppButton({ appId, className }: { appId: stri
   const utils = trpc.useContext();
 
   const mutation = useAddAppMutation(null, {
-    onSuccess: () => {
+    onSuccess: (data: any) => {
       //TODO: viewer.appById might be replaced with viewer.apps so that a single query needs to be invalidated.
       utils.invalidateQueries(["viewer.appById", { appId }]);
       utils.invalidateQueries(["viewer.apps", { extendsFeature: "EventType" }]);
-      showToast(t("app_successfully_installed"), "success");
+      if (data.showToast !== undefined && data.showToast) {
```

Comment:
This is to fix the false positive "App successfully installed" toast message
cc: @hariombalhara

Reply:
I think we can remove `data.showToast !== undefined`